spec/views/v1/collections/_collection.json.jbuilder_spec.rb

```diff
@@ -5,12 +5,11 @@
     render partial: 'v1/collections/collection', locals: { collection: Purl.first }
     expect(JSON.parse(rendered)).to include(
       'druid' => 'druid:ee111ff2222',
-      'true_targets' => ['SearchWorksPreview'],
       'catkey' => ''
     )
   end
-  it 'always returns "SearchWorksPreview"' do
-    render partial: 'v1/purls/purl', locals: { purl: Purl.first }
+  it 'always returns "SearchWorksPreview" for non deleted Purls' do
+    render partial: 'v1/collections/collection', locals: { collection: Purl.find(6) }
```

Comment:
It's not clear to me why Purl `id=6` is the answer here. The test fixture for 6 says "6 should have no collection". Maybe add a comment?